Menu/Menu.js

```diff
@@ -1,10 +1,20 @@
 
+
+
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  // Toggle the "menu--open" class on your menu refence.
+  menu.classList.toggle('menu--open'); //not sure this line of code is correct but we can move on
+  console.log('toggle');
 }
 
+
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
 // create a reference to the ".menu-button" class
-const menuButton;
+const menuButton = document.querySelector('.menu-button');
 // Using your menuButton reference, add a click handler that calls toggleMenu
+menuButton.addEventListener('click', () => {
```

Comment:
You could forego the wrapping function and just use `toggleMenu` as your callback here:
```javascript
 menuButton.addEventListener('click', toggleMenu);
```